spec.html

```diff
@@ -16649,9 +16744,12 @@ <h1>Static Semantics: Early Errors</h1>
         </emu-note>
       </emu-clause>
 
-      <emu-clause id="sec-static-semantics-sv" oldids="sec-string-literals-static-semantics-stringvalue,sec-additional-syntax-string-literals-static-semantics" type="sdo" aoid="SV">
+      <emu-clause id="sec-static-semantics-sv" oldids="sec-string-literals-static-semantics-stringvalue,sec-additional-syntax-string-literals-static-semantics" type="sdo">
         <h1>Static Semantics: SV</h1>
-        <p>A string literal stands for a value of the String type. SV produces String values for string literals through recursive application on the various parts of the string literal. As part of this process, some Unicode code points within the string literal are interpreted as having a mathematical value, as described below or in <emu-xref href="#sec-literals-numeric-literals"></emu-xref>.</p>
+        <dl class="header">
+          <dt>description</dt>
+          <dd><p>A string literal stands for a value of the String type. SV produces String values for string literals through recursive application on the various parts of the string literal. As part of this process, some Unicode code points within the string literal are interpreted as having a mathematical value, as described below or in <emu-xref href="#sec-literals-numeric-literals"></emu-xref>.</p></dd>
```

Comment:
```suggestion
          <dd>A string literal stands for a value of the String type. SV produces String values for string literals through recursive application on the various parts of the string literal. As part of this process, some Unicode code points within the string literal are interpreted as having a mathematical value, as described below or in <emu-xref href="#sec-literals-numeric-literals"></emu-xref>.</dd>
```

We don't wrap these in `<p>` elsewhere.

Reply:
That's because these descriptions are not normally an entire paragraph of text. Wrapping in `<p>` causes this part of the description to be set off in its own paragraph, with the "The syntax-directed operation X takes Y." and "It is defined piecewise over the following productions" parts as whole paragraphs before and after the description. That read better to me, which is why I've put this in `<p>`.

Reply:
It seems weird that it's typically a text node which is inserted as a child of the summary `<p>`, but when it's a `<p>`, it becomes a sibling.

Reply:
Does it? It seems like the obvious behavior to me.

Reply:
Discussed offline; @michaelficarra said he was OK with this.

Reply:
[This is a new "conversation", but GitHub doesn't recognize that.]
```suggestion
          <dd>
            <p>A string literal stands for a value of the String type. SV produces String values for string literals through recursive application on the various parts of the string literal. As part of this process, some Unicode code points within the string literal are interpreted as having a mathematical value, as described below or in <emu-xref href="#sec-literals-numeric-literals"></emu-xref>.</p>
          </dd>
```
The only other example of `<p>` within `<dd>` is at `ResolveExport`, which formats it this way. (Of course, it has to, because there are 2 such `<p>` elements. Still, given the difference in rendering, I think it makes sense to format the 1 `<p>` case more like the 2 `<p>` case than the 0 `<p>` case.)